test/sasdataloader/test/utest_generic_file_reader_class.py

```diff
@@ -0,0 +1,46 @@
+"""
```

Comment:
These tests have been added to test/sasdataloader/test/utest_extension_registry.py